server/handlers/handlers.go

```diff
@@ -361,16 +369,23 @@ func (ah *ActivityHandler) handleUpdatedActivity(activityIds msteams.ActivityIds
 
 	_, appErr := ah.plugin.GetAPI().UpdatePost(post)
 	if appErr != nil {
-		ah.plugin.GetAPI().LogError("Unable to update post", "post", post, "error", appErr)
-		return
+		if strings.Contains(appErr.Error(), "Unable to get the post.") {
```

Comment:
Same here, I prefer to check the `appErr.Id` to avoid problems with translations.